nixos/modules/config/users-groups.nix

```diff
@@ -686,6 +686,8 @@ in {
 
     environment.profiles = [
       "$HOME/.nix-profile"
+      "\${XDG_STATE_HOME}/nix/profile"
```

Comment:
Why is the XDG_STATE_HOME line using a different environment expansion syntax?

Reply:
@bjornfor wdym?

Reply:
$HOME is actually special-cased and converted to `${HOME}` in https://github.com/NixOS/nixpkgs/blob/f99e5f03cc0aa231ab5950a15ed02afec45ed51a/nixos/modules/config/system-environment.nix#L75

Reply:
Ok, thanks.